tests/testthat/test-sitevisitpeak.R

```diff
@@ -4,17 +4,70 @@ wd <- getwd()
 setwd(dir = tempdir())
 
 context("testing sitevisitpeak")
+
 test_that("sitevisitpeak examples work",{
   library(jsonlite)
   
   data <- fromJSON(system.file('extdata','sitevisitpeak','sitevisitpeak-example.json', package = 'repgen'))
-  expect_is(sitevisitpeak(data,'html', 'Author Name'), 'character')
+  expect_is(sitevisitpeak(data, 'html', 'Author Name'), 'character')
```

Comment:
Yes, you can. This is actually passing "html" as the author name and the author name is ignored.